Backport the duplicate test case fix from upstream's broader lint cleanup.
https://github.com/statsmodels/statsmodels/commit/f49757024b9c12737b9a54a642f25298e4af3b2f

--- a/statsmodels/stats/tests/test_moment_helpers.py
+++ b/statsmodels/stats/tests/test_moment_helpers.py
@@ -104,9 +104,9 @@
     assert_almost_equal(mvsk2mc(mc2mvsk(test_vals).T).T, test_vals)
 
 
-@pytest.mark.parametrize('func_name', ['cum2mc', 'cum2mc', 'mc2cum', 'mc2mnc',
+@pytest.mark.parametrize('func_name', ['cum2mc', 'mc2cum', 'mc2mnc',
                                        'mc2mvsk', 'mnc2cum', 'mnc2mc',
-                                       'mnc2mc', 'mvsk2mc', 'mvsk2mnc'])
+                                       'mvsk2mc', 'mvsk2mnc'])
 def test_moment_conversion_types(func_name):
     # written in 2009
     # TODO: why did I use list as return type?
